frontend/amundsen_application/static/js/components/Lineage/Graph/chart.ts

```diff
@@ -327,7 +327,11 @@ export const buildNodes = (g, targetNode, nodes, onClick) => {
     .append('text')
     .attr('dy', NODE_LABEL_Y_OFFSET)
     .attr('text-anchor', 'middle')
-    .text((d, idx) => (idx !== 0 && d.data.data.name ? d.data.data.name : ''));
+    .text((d, idx) =>
+      idx !== 0 && d.data.data.name
+        ? d.data.data.schema + '.' + d.data.data.name
+        : ''
+    );
```

Comment:
I would extract a function from this, maybe 'getNodeName'

Reply:
Good suggestion, I've extracted it into `getNodeLabel` function